heron/ui/src/python/main.py

```diff
@@ -134,14 +136,15 @@ def main():
 
   # log additional information
   command_line_args = vars(parsed_args)
-  address = socket.gethostbyname(socket.gethostname())
+
+  address = '0.0.0.0' if command_line_args['address'] is None else command_line_args['address']
```

Comment:
@kramasamy honestly I don't know right now. I will test it later. My assumption is from https://github.com/tornadoweb/tornado/blob/7ec71d46a83eade88f6d588f8f568f9759676c89/tornado/netutil.py#L111.